dls-controls/malcolmjs
# Feature/method info #244

## Description

Added info pane for methods and their parameters. Also some minor style tweaks and bugfixes for methods.

## Testing instructions

Add a set up instructions describing how the reviewer should test the code
- Review code
- Check Travis build
- Review changes to test coverage
- navigate to localhost:3000/gui/PANDA
- click on the info icon next to the save button and view the method info
- click on the info icon next to "design" either in the "save" section of the parent pane or the "input parameter types" section of the info pane, and view the method parameter info pane 

## Agile board tracking

connect to #244 
connect to #245 
closes #244 
closes #245
